### PR TITLE
Added seven datasets

### PR DESCRIPTION
**Pull request by Extord1108**
AGGC, DHMC-Kidney, DHMC-Lung, SegPath, Prostate Fused-MRI-Pathology, Tolkach Y. et al. and UPENN-GBM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Dataset name | Organs | Staining | Link | Size | Data | Task | WSI/Patch | Other
...
 ACROBAT 2022 [66] | Breast | Multiple (IHC, H&E) | [data](https://acrobat.grand-challenge.org/), [paper](https://openreview.net/pdf?id=5TTocO2VI3r) | Train: 750 train; Valid: 100; Test: 300 | images (1 H&E match to 1-4 IHC) + landmarks | registration | wsi | 40x - Hamamatsu | 2022
 ADP [2] | multiple | multiple (most H&E) | [data](https://www.dsp.utoronto.ca/projects/ADP/), [github](https://github.com/mahdihosseini/ADP), [paper](https://openaccess.thecvf.com/content_CVPR_2019/papers/Hosseini_Atlas_of_Digital_Pathology_A_Generalized_Hierarchical_Histological_Tissue_Type-Annotated_CVPR_2019_paper.pdf) | Train: 14.134, Valid: 1767, Test: 1767 (100 wsi) | images + 57 hierarchical HTTs (histological tissue type) | multi-label (3) classification (hierarchy) | patch (1088x1088) | 40x - Huron TissueScope LE1.2 WSI | 2019
+AGGC | prostate | H&E | [data](https://aggc22.grand-challenge.org/AGGC22/) | Subset 1: train 105, test 45; Subset2: train 37 ,test 16; Subset3: train 144, test 67 | images scaned by different scanner | seg + gleason grading | wsi | Subset1 and Subset2: Akoya Biosciences Scanner, Subset3: each specimen is scanned by multiple scanners |
```
Comment (maduc7) [inline README.md:23]: I would modify as followed:
AGGC | prostate | H&E | [data](https://aggc22.grand-challenge.org/AGGC22/), [paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4172090) | Subset 1: train 105, test 45; Subset2: train 37 ,test 16; Subset3: train 144, test 67 | images + binary masks | seg + gleason grading | wsi | 20x - Subset1 and Subset2: Akoya Biosciences Scanner, Subset3: each specimen is scanned by multiple scanners | 2022

File: README.md [hunk 2]
```diff
@@ -89,23 +92,27 @@ PAIP2020 [41] | Colon | H&E | [data](paip2020.grand-challenge.org/), [github](ht
...
 TissueNet | Uterine cervix | H&E | [data](https://www.drivendata.org/competitions/67/competition-cervical-biopsy/page/254/), [github](https://github.com/drivendataorg/tissuenet-cervical-biopsies) | 1,016 WSIs; 5,926 patches (1200x1200 px) | images + annotation + metadata + labels | classi (4) | wsi + patches | MIRAX, Aperio, Hamamatsu | 2020
 TNBC [50] | Breast | H&E | [data](https://peterjacknaylor.github.io/data/), [data](https://drive.google.com/drive/folders/1taB8boGyycjV4X1a2vCIAV9fwMxFSS41), [paper](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=8438559) | 50 images, 4022 cells (11 patients) | images + nuclei seg + label | seg + classi | patch (512x512) | 40x - Philips Ultra Fast Scanner (Curie Inst.) | 2019
+Tolkach Y. et al. [84] | oesophageal adenocarcinomas | H&E | [data](https://zenodo.org/records/7548828), [paper](https://www.thelancet.com/journals/landig/article/PIIS2589-7500(23)00027-4/fulltext) | UKK1: 22 wsi (20 patients); WNS: 62 wsi (15 patients); CHA: 214wsi (69 patients); TCGA:22 wsi (22 patients) | images + label | classi | patch(256x256) | 40x - Nanozoomer S360 | 2023
```
Comment (maduc7) [inline README.md:110]: Is it slides or patches or both? Because you wrote patch and also wsi.

**Review by maduc7 — CHANGES_REQUESTED** [review]

**Comment by maduc7** [comment]
Great thanks a lot for your contribution! It looks really good, I just left two minor comments.

File: README.md (hunk 2 above)
Comment (Extord1108) [reply README.md:110]: Only patches, I will add the number of them.

**Review by maduc7 — APPROVED** [review]

**Pull request merged**